docker/bookkeeper/Dockerfile

```diff
@@ -33,12 +42,34 @@ RUN set -x \
     && gpg --import KEYS \
     && gpg --batch --verify "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz" \
     && tar -xzf "$DISTRO_NAME.tar.gz" \
-    && cp -r bookkeeper-all-${BK_VERSION}/* /opt/bookkeeper/ \
+    && mv bookkeeper-all-${BK_VERSION}/ /opt/bookkeeper/ \
     && rm -rf "bookkeeper-all-${BK_VERSION}" "$DISTRO_NAME.tar.gz" "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz.sha512" \
```

Comment:
This should delete all downloaded archives as well as the unpacked. Was there something wrong with that step?

Reply:
Havent changed the delete part

Reply:
Yes, it hasn't changed I get. What am I actually looking for is an explanation why we have duplicate bookies in the first place. My initial thought was that it's because we download bookie artifacts and keep them for whatever reason. So if this is the case then we need to discard whatever we download and continue to rely on the official bookie image as the base image.

Reply:
In the official bk image, we are using bk-server. It doesnt have below  libaries that are there in bk-full

```com.google.code.findbugs-jsr305-3.0.2.jar
org.apache.bookkeeper-bookkeeper-benchmark-4.11.1.jar
io.dropwizard.metrics-metrics-core-3.1.0.jar
io.dropwizard.metrics-metrics-jvm-3.1.0.jar
io.dropwizard.metrics-metrics-graphite-3.1.0.jar
org.apache.bookkeeper.stats-codahale-metrics-provider-4.11.1.jar```